packages/pixeloven-core/env/src/Utils.ts

```diff
@@ -6,10 +6,13 @@ import { Mode, Name, Target, UtilOptions } from "./types";
  */
 export function getUtils<T extends UtilOptions>(options: T) {
     function ifType<Y, N>(isType: boolean, value?: Y, alternate?: N) {
-        const tValue = value || false;
-        const tAlternate = alternate || false;
```

Comment:
I didn't like this it was confusing and not quite accurate since if alternate was 0 this would eval to false.